tests/rt.conf

```diff
@@ -502,7 +503,7 @@ COMPILE | s2sw_pdlib | gnu | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugw
 RUN | cpld_control_pdlib_p8                             | + hera hercules                           | baseline |
 
 COMPILE | s2sw_pdlib_debug | gnu | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON -DDEBUG=ON | + hera hercules | fv3 |
-RUN | cpld_debug_pdlib_p8                               | + hera hercules                           | baseline |
+RUN | cpld_debug_pdlib_p8                               | + hercules                           | baseline |
```

Comment:
I thougth this was working on hera now?  Can this be added back @RatkoVasic-NOAA

Reply:
@JessicaMeixner-NOAA I updated it to run it on Hera. I'll run it this way, let's see what hatches.